### PR TITLE
Exclude openshift clusters in the image-loader

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Exclude openshift clusters in the image-loader & fix the pushing of images.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: api/cmd/kubermatic-controller-manager/options.go [hunk 1]
```diff
@@ -107,7 +108,7 @@ func newControllerRunOptions() (controllerRunOptions, error) {
 	flag.StringVar(&c.oidcIssuerClientSecret, "oidc-issuer-client-secret", "", "OpenID client secret")
 	flag.BoolVar(&c.log.Debug, "log-debug", false, "Enables debug logging")
 	flag.StringVar(&c.log.Format, "log-format", string(kubermaticlog.FormatJSON), "Log format. Available are: "+kubermaticlog.AvailableFormats.String())
-	flag.StringVar(&c.kubermaticImage, "kubermatic-image", "quay.io/kubermatic/api", "The location from which to pull the Kubermatic image")
+	flag.StringVar(&c.kubermaticImage, "kubermatic-image", resources.DefaultKubermaticImage, "The location from which to pull the Kubermatic image")
```
Comment (alvaroaleman) [inline api/cmd/kubermatic-controller-manager/options.go:111]: This flag has no effect or am I missing?

Comment (mrIncompetent) [reply api/cmd/kubermatic-controller-manager/options.go:111]: You introduced that flag in https://github.com/kubermatic/kubermatic/pull/3519

Comment (alvaroaleman) [reply api/cmd/kubermatic-controller-manager/options.go:111]: Oh, thought it was a flag on the image loader

/lgtm
/approve

**Pull request merged**